src/fontra/views/editor/panel-designspace-navigation.js

```diff
@@ -796,9 +821,12 @@ export default class DesignspaceNavigationPanel extends Panel {
       let layerMessage = "";
       if (layer !== undefined && deleteLayerCheckBox.checked) {
         delete glyph.layers[source.layerName];
-        layerMessage = " and layer";
+        layerMessage = ` ${translate("sidebar.designspace-navigation.undo.and-layer")}`;
```

Comment:
I think the leading space should be part of the string in the spreadsheet, like we do elsewhere. It depends on the language whether it should be there or not.

Reply:
Thanks for the info. I changed it.